org-journal.el

```diff
@@ -1880,12 +1879,17 @@ Only one recipient is supported.  ")))
                    'before-save-hook)
       (save-buffer))))
 
-;; Setup encryption by default
-;;;###autoload
-(add-hook 'org-journal-mode-hook
-          (lambda () (add-hook org-journal-encrypt-on
-                               'org-journal-encryption-hook
-                               nil t)))
+(defun org-journal-add-encryption-hook ()
```

Comment:
I think the function name should not end on `-hook`, so as to not confuse people with the hook variable itself.

Reply:
Changed the function name to `org-journal-default-enable-encryption`, but I think the naming of the encryption functions should be rethought a little more in a separate issue/PR.

Reply:
I agree! But as you say, I'll leave that for another issue or pull request.